valet/valet_suite_test.go

```diff
@@ -658,7 +667,7 @@ var _ = Describe("Archive MINKnow files", func() {
 							return
 						}
 
-						if len(contents) >= len(expected) {
+						if len(contents) >= len(expectedArchived) {
```

Comment:
It's not a test pass, it just says stop loading now because we have at least the number of files where the test could potentially pass. If more than expected were loaded somehow, we still want to stop because we know we can fail early.